services/src/server.rs

```diff
@@ -73,7 +73,7 @@ pub async fn start_server(
                 shutdown_rx,
                 static_files_dir,
                 bind_address,
-                InMemoryContext::default(),
+                InMemoryContext::new_with_data().await,
```

Comment:
Currently, we have a fixed set of data and later on, we will implement adding data, right?

Reply:
Adding data will come, yes. The idea was to have some data sets already available for testint and as the in memory context is initially empty I add the default data sets here. For Postgres we would only do this once/ad-hoc I guess because then they are stored permanently.